src/components/Leaderboard.tsx

```diff
@@ -35,37 +36,14 @@ const difficultyColor = (difficulty: string): string => {
   }
 };
 
-const headers = (isGitHub: boolean): Header[] => {
-  if (isGitHub) {
-    return [
-      { key: 'rank', label: 'Rank' },
-      { key: 'username', label: 'Username' },
-      { key: 'totalCommits', label: 'Commits' },
-      { key: 'totalPrs', label: 'PRs' },
-      { key: 'followers', label: 'Followers' },
-    ];
-  }
-  return [
-    { key: 'rank', label: 'Rank' },
-    { key: 'username', label: 'Username' },
-    { key: 'totalSolved', label: 'Total' },
-    { key: 'easySolved', label: 'Easy' },
-    { key: 'mediumSolved', label: 'Medium' },
-    { key: 'hardSolved', label: 'Hard' },
-  ];
-};
-
 interface LeaderboardProps {
   data: GitHubStats[] | LeetCodeStats[];
-  type: LeaderboardType;
+  type?: LeaderboardType;
```

Comment:
why is this optional?

Reply:
Forgot to get rid of it, made it optional so that the website wouldn't break during development